### ISSUE TITLE
The Windows and Linux packages don't provide out of the box firewall configuration.

### ISSUE THREAD

**Issue by enternoescape** [enternoescape/opendct#14]
In Windows it is normal for an installer to also create Firewall exceptions, so I will build the functionality into the installation. In Linux it is normal/preferred to configure these settings yourself.

CentOS 7 and Fedora 22 use firewalld out of the box. We can create an xml file and add it to the package located at /etc/firewalld/services/opendct.xml so that enabling the exceptions will be as easy as typing the code block below.
```
firewall-cmd --get-default-zone
firewall-cmd --permanent --zone=<default_zone> --add-service=opendct
firewall-cmd --reload
```

Ubuntu 14.04 uses ufw by default to make configuration of the firewall easier, but it's not enabled out of the box. Their argument is the default installation isn't open enough to warrant it being enabled. I will add a script to enable/disable the needed ports since appending a new service to /etc/services could be problematic; especially if the created entries are modified and the package is being removed. The commands needed to enable the firewall for OpenDCT are in the code block below.
```
ufw allow 7818/tcp
ufw allow 9000-9100/tcp
ufw allow 8271/udp
ufw allow 8100-8500/udp
```

**Comment by enternoescape** [comment]
I just noticed for the first time that the SageTV discovery port conflicts with the RTP receiving port range. I'm going to fix that too. You can manually fix it by changing rtsp.port\_low=8100 to 8300.

**Comment by enternoescape** [comment]
I changed the default port range for RTP traffic to 8300-8500 so we don't have any more conflict with the discovery port. Fedora 22 and CentOS 7 properly provide a firewalld service and Ubuntu has a very simple script to apply firewall settings and remove them. My only hangup right now is Windows. I had to use a batch file to apply the firewall settings because the FirewallException scheme used in WiX was not working correctly. This will be revisited if it proves to be a problem.
